src/seedu/addressbook/ui/Formatter.java

```diff
@@ -0,0 +1,44 @@
+package seedu.addressbook.ui;
+
+public class Formatter {
+
+    /**
+     * A decorative prefix added to the beginning of lines printed by
+     * AddressBook
+     */
+    private static final String LINE_PREFIX = "|| ";
+
+    /** A platform independent line separator. */
+    private static final String LS = System.lineSeparator();
+
+    private static final String DIVIDER = "===================================================";
+
+    /** Format of indexed list item */
+    private static final String MESSAGE_INDEXED_LIST_ITEM = "\t%1$d. %2$s";
+
+    public String getLinePrefix() {
+        return LINE_PREFIX;
+    }
+
+    public String getLs() {
+        return LS;
+    }
+
+    public String getDivider() {
+        return DIVIDER;
+    }
+
+    public String getMessageIndexedListItem() {
+        return MESSAGE_INDEXED_LIST_ITEM;
+    }
+
+    /**
+     * Formats a string as a viewable indexed list item.
+     *
+     * @param visibleIndex
+     *            visible index for this listing
+     */
+    public String getIndexedListItem(int visibleIndex, String listItem) {
+        return String.format(MESSAGE_INDEXED_LIST_ITEM, visibleIndex, listItem);
```

Comment:
Perhaps more formatting-related stuff (e.g. `DIVIDER`) can be moved to the `Formatter`? May be even things like this should be moved to the Formatter? Note that it may not always be clear if something belongs in the Formatter class or the TextUi class. It depends on how you define the responsibility of the Formatter class.